### DISCUSSION TITLE
iFrame integration security issue showing private key

### DISCUSSION THREAD

**Discussion by radar1975** [apache/superset#17538] [Q&A / Help]
I just figure out that when you embed a chart iFrame into a Web app Apache Superset inject some database connection information into the Web page source. This is a major security issue and i'm wondering if someone have found a solution for that.

I have attached a picture of the source code of the iframe. You can see that the private key of the database connection is exposed. **This is a none sens.**

![Capture](https://user-images.githubusercontent.com/1986318/143265015-a06e3b6c-840a-47dd-8c2b-bf26448f204f.PNG)

**Comment by chrisjung-dev** [comment]
At least , works for you.
